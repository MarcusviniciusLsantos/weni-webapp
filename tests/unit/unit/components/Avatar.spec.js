import { shallowMount } from '@vue/test-utils';
import Avatar from '@/components/Avatar.vue';

describe('Avatar.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(Avatar, {
      mocks: {
        $t: () => '',
      },
    });
  });

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it('imageBackground()', async () => {
    await wrapper.setProps({
      imageUrl: 'top',
    });

    expect(wrapper.vm.imageBackground).toEqual("background-image: url('top')");
  });
});
