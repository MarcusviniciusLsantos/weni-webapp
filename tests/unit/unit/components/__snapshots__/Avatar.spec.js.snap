// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Avatar.vue > renders correctly 1`] = `
<div
  class="weni-avatar weni-avatar--size-md"
  data-v-69299821=""
>
  <img
    data-v-69299821=""
    height="100%"
    src="/src/assets/default-avatar.svg"
    width="100%"
  />
</div>
`;
